.github/workflows/ci.yml

```
@@ -24,13 +24,17 @@ jobs:
       contents: "read"
     steps:
       - uses: actions/checkout@v4
+
+      - uses: DeterminateSystems/flake-checker-action@main
+        if: ${{ always() }}
```

```suggestion
        if: success() || failure()
```

is preferred over `always()`, since `always()` will run even in the face of a cancellation.

---

(below as well)